hack/components/bump-linux-bridge.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -xe
```

Comment:
others have -o pipefail as well if you want to add here as well

Reply:
I agree, we should keep all bump scripts similar as possible for better maintenance and readability